_rules/document-has-headings-047fe0.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 047fe0
+name: Document has headings
+rule_type: atomic
+description: |
+  This rule checks that each section of content starts with a heading
+accessibility_requirements:
+  wcag-technique:H69: # Providing heading elements at the beginning of each section of content
+		forConformance: false
+		failed: not satisfied
+		passed: satisfied
+		inapplicable: satisfied
+input_aspects:
+	- DOM Tree
+	- CSS styling
+authors:
+	- Jean-Yves Moyen
+	- Anne Thyme Nørregard
+---
+
+## Applicability
+
+This rule applies to any [document](#https://www.w3.org/TR/dom/#concept-document) where the [document element](#https://www.w3.org/TR/dom/#document-element) is an HTML `html` element.
+
+## Expectations
+
+Each [section of content](#section-of-content) in the test target either starts with or is preceded by an element with a [semantic role](#semantic-role) of `heading` which is [included in the accessibility tree](#included-in-the-accessibility-tree).

```

Comment:
How do you define "starts with" and "preceded by"? Is it in the focusable order? In the flat tree? In the accessibility tree?